README.md

```diff
@@ -0,0 +1,52 @@
+
+# Hello Microverse
+
+This is project is going to help us to master github flow and get used with linters for checking errors in our css ans Html files.
+
+## Materials required for validation
+
+- npm install --save-dev hint@6.x
+- npx hint .
+- npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x       stylelint-csstree-validator@1.x
+
+
+
+## Built With
+
+- HTML
+- CSS
+
+
+## Get Started
+To get a local copy up and running follow these simple example steps.
+
+Use this command to have the copy of the project in your Current directory
+-  git clone git@github.com:moise10r/Hello_World.git.
+
+## Authors
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/moise10r)
+- Twitter: [@twitterhandle](https://twitter.com/MRushanika)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/nganulo-rushanika-mo%C3%AFse-626139197/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] **OPTIONAL**
Please keep in mind that it could leave a bad impression on future viewers if they find a non-working link attached in the README file. It'd be great if you fixed the MIT link.